### PR TITLE
Madara: fix pages empty for some source

### LINKED ISSUE #9863: ManhuaUS - pages not found
### Source information

ManhuaUS 1.2.14

### Steps to reproduce

Look at chapter in manhua on source.

### Expected behavior

Read and download normally

### Actual behavior

Extension isn't detecting pages in the latest chapters

### Tachiyomi version

0.12.3

### Android version

Android 9

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #9864: manga1st - pages not found
### Source information

manga1st 1.2.12

### Steps to reproduce

Look at chapter in manhua on listed source.

### Expected behavior

Read and download normally.

### Actual behavior

Extension isn't detecting pages in the latest chapters

### Tachiyomi version

0.12.3

### Android version

Android 9

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #9865: MangaGreat - pages not found
### Source information

MangaGreat 1.2.12

### Steps to reproduce

Look at chapter in manhua on listed source.

### Expected behavior

Read and download normally.

### Actual behavior

Extension isn't detecting pages in the latest chapters

### Tachiyomi version

0.12.3

### Android version

Android 9

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #9866: manga347 - pages not found
### Source information

manga347 1.2.14

### Steps to reproduce

Look at chapter in manhua on listed source.

### Expected behavior

Read and download normally.

### Actual behavior

Extension isn't detecting pages in the latest chapters

### Tachiyomi version

0.12.3

### Android version

Android 9

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Riztard**
Closes #9863
Closes #9864 
Closes #9865
Closes #9866

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by Riztard** [comment]
going to check again if this change was from madara update or just many source did the same thing

File: multisrc/overrides/madara/manga68/src/Manga68.kt [hunk 1]
```diff
@@ -2,6 +2,4 @@ package eu.kanade.tachiyomi.extension.en.manga68
 
 import eu.kanade.tachiyomi.multisrc.madara.Madara
 
-class Manga68 : Madara("Manga68", "https://manga68.com", "en") {
-    override val pageListParseSelector = "div.page-break, div.text-left p"
-}
+class Manga68 : Madara("Manga68", "https://manga68.com", "en")
```
Comment (arkon) [inline multisrc/overrides/madara/manga68/src/Manga68.kt:5]: You should be able to just delete some of these overrides if they're no longer actually overriding anything.

Comment (Riztard) [reply multisrc/overrides/madara/manga68/src/Manga68.kt]: true, im just too lazy if i need to create new override file

**Review by arkon — COMMENTED** [review]
You didn't need to delete the ManhuaBox icons.

**Comment by Riztard** [comment]
> You didn't need to delete the ManhuaBox icons.

it has 2 icon one on the outside res folder

**Pull request merged**